ADAL/src/cache/ADTokenCache.m

```diff
@@ -69,6 +69,45 @@ - (void)dealloc
     SAFE_ARC_SUPER_DEALLOC();
 }
 
+- (id)cacheObjectForKey:(id<NSCopying>)key
+{
+    id object = nil;
+    @synchronized(self)
```

Comment:
As long as it takes me to hit the 'merge' button. Getting out a release is another matter though.